sdk/identity/azure-identity-broker/azure/identity/broker/_utils.py

```diff
@@ -0,0 +1,109 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from typing import List, Optional
+import os
+import functools
+import logging
+from contextvars import ContextVar
+from azure.core.exceptions import ClientAuthenticationError
+
+
+class EnvironmentVariables:
```

Comment:
I don't think we need to copy the entire thing. Looks like we only use  `AZURE_IDENTITY_DISABLE_MULTITENANTAUTH`, so can't just have that be a singular constant in this file